src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -250,5 +250,11 @@
     "DESCRIPTIONS": [
       "Whenever you draw a Curse, deal #b%1$s damage to ALL enemies."
     ]
+  },
+  "stsjorbsmod:WitherPower": {
+    "NAME": "Wither",
+    "DESCRIPTIONS": [
+      "Intangible reduces HP loss and damage taken by 1 less."
```

Comment:
"1" should probably be `#b%1$s` instead to account for stacking behavior (you're already filling in `amount` in `updateDescription`)